docs/src/modules/ROOT/pages/optimization-algorithms/overview.adoc

```diff
@@ -565,6 +565,58 @@ Terminates as soon as a feasible solution has been discovered.
 
 This `Termination` is usually combined with other terminations.
 
+[#unimprovedBestSolutionTermination]
+==== `UnimprovedBestSolutionTermination`
+
+`UnimprovedBestSolutionTermination` terminates when the solver cannot improve the best solution in an estimated time interval.
+The estimation of the termination interval is based on the last time
+the solver improved the solution and how much time has passed without any improvements.
```

Comment:
Are we using time? I thought we were using move counts.

Reply:
We estimate time using the move evaluation speed and move count. However, I am considering using time instead. This would solve a corner case I couldn't address while maintaining similar behavior to the current implementation.